src/ghga_datasteward_kit/s3_upload/utils.py

```diff
@@ -150,7 +197,7 @@ def check_adjust_part_size(config: LegacyConfig, file_size: int):
             )
 
     if part_size != config.part_size * 1024**2:
-        LOGGER.info(
+        LOG.info(
```

Comment:
```suggestion
        LOG.warning(
```

Reply:
When does that happen?

Reply:
Two cases:
1) Either the configured part size is not within what's supported in Amazon's S3 spec, so the value is clamped
2) Very large file and encryption would lead to more than the supported 10.000 parts with the configured part size, so we choose the closest larger part size that would yield at most 10.000 parts